CONTRIBUTING.md

```diff
@@ -62,28 +62,34 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Collaborator
+# Becoming a Member and Collaborator
 
-All contributors who land a non-trivial contribution should be on-boarded in a timely manner,
+To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, 
+file issues and pull requests, and provide insight via GitHub. Request to become an Observer by 
```

Comment:
Yeah, this was meant as a guideline of possible ways -to- contribute. I don't want to set requirements.